portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -51,4 +51,47 @@ private String convertToJson(List messages) {
       Gson gson = new Gson();
       return gson.toJson(messages);
     }
+
+    @Override
+    public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+      // Get the input from the form.
+      String text = getParameter(request, "word-input", "");
+      boolean cut = Boolean.parseBoolean(getParameter(request, "cut", "false"));
+      boolean add = Boolean.parseBoolean(getParameter(request, "add", "false"));
+      boolean lowerCase = Boolean.parseBoolean(getParameter(request, "lowerCase", "false"));
+      
+
+    // Storing comments in their respective bins.
+    if (cut) {
+      text = text.substring(1);
+    }
+
+    if (add) {
+      text = text + 'a';
+    }
+
+    if (lowerCase) {
+      text = text.toLowerCase();
+    }
+
+    Entity taskEntity = new Entity("Task");
+    taskEntity.setProperty("comment", text);
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    datastore.put(taskEntity);
+
+    // Respond with the result.
+    response.setContentType("text/html;");
+    response.getWriter().println(text);
+  }
+  /**
```

Comment:
nit: add empty line before function declaration

Reply:
Makes sense, fixed all the issues now. Will keep trying to have less nits in the future! :)